bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/ZprimeTo2ChiTo2L/ZprimeTo2ChiTo2L_extramodels.dat

```diff
@@ -0,0 +1 @@
+UMSSM.tar
```

Comment:
Please provide the corresponding UFO. It is not on our model page.

Reply:
could you provide me the path? any write permission needed?

Reply:
Upload it in public lxplus repo, we can handle it